src/jit/codegencommon.cpp

```diff
@@ -2692,11 +2692,15 @@ void CodeGen::genExitCode(BasicBlock* block)
 
 void CodeGen::genJumpToThrowHlpBlk(emitJumpKind jumpKind, SpecialCodeKind codeKind, GenTreePtr failBlk)
 {
-    if (!compiler->opts.compDbgCode)
-    {
-        /* For non-debuggable code, find and use the helper block for
-           raising the exception. The block may be shared by other trees too. */
+    // For non-debuggable code, find and use the helper block for
+    // raising the exception. The block may be shared by other trees too.
+    //
+    // Otherwise, the code to throw the exception will be generated inline,
+    // and we will jump around it in the normal non-exception case
+    bool useExnThrowingBlock = !compiler->opts.compDbgCode;
```

Comment:
I'm ok with your changes to codegencommon.cpp as "cleanup" changes. But you didn't make any functional changes, so it might be better to just not change the file at all, for simplicity.